src/libs/antares/study/parameters.cpp

```diff
@@ -1155,6 +1138,15 @@ static bool SGDIntLoadFamily_Legacy(Parameters& d,
         return true;
     }
 
+    if (key == "initial-reservoir-levels") // ignored since 9.2
+    {
+        if (version >= StudyVersion(9,2))
+            logs.warning() << "Option initial-reservoir-levels is deprecated, please remove it from the study";
+        else if (value == "hot start")
+            logs.warning() << "Hydro hot start not supported with this solver";
```

Comment:
Please use a version <9.2

Reply:
`if version < 9.2 && value == hot start` ?

Reply:
yes, complete the warning message at l. 1146.